fw/src/app/core_comms.cpp

```diff
@@ -141,7 +153,10 @@ static PT_THREAD(run10ms(struct pt* thread))
             serial_getLine(PORT_RPI, core_comms_data.line);
             core_comms_parseLine(core_comms_data.line);
             char* resp = (char*) malloc(SERIAL_MESSAGE_SIZE);
-            sprintf(resp, "{\"system_state\": %d}\n", system_state_getState());
+            sprintf(resp,
+                    "{\"system_state\": %d, \"belts_state\": %d}\n",
+                    system_state_getState(),
+                    belts_getState());
```

Comment:
Rudy will eventually need `depositor_state`, might be good to make the change here

Reply:
he can monitor system state no?